README.md

```diff
@@ -55,12 +55,12 @@ See the official [React Native website](https://facebook.github.io/react-native/
 
 - You can run React Native Windows UWP apps only on Windows 10 devices
    - React Native Windows -[current](https://github.com/microsoft/react-native-windows/tree/master/current) supports Windows 10 SDK > [14393](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive) and Windows 8.1
-   - React Native Windows -[vnext](https://github.com/microsoft/react-native-windows/tree/master/vnext) currently supports Windows 10 SDK > [16299](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
+   - React Native Windows -[vnext](https://github.com/microsoft/react-native-windows/tree/master/vnext) currently supports Windows 10 SDK > [15063](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
```

Comment:
It should be >= 15063, and it also apply to other place too.

Reply:
Updated.